docs(cloudevents): try cloudevents-nodejs with kn (#2827)
knative/docs: docs/serving/samples/cloudevents/cloudevents-nodejs/README.md

- [Docker](https://www.docker.com) installed and running on your local machine,
  and a Docker Hub account configured (we'll use it for a container registry).

## The Sample Code

> In `index.js`, you will see two key functions for the different modes of operation:

Reviewer: ```suggestion
In the `index.js` object, you will see two key functions for the different modes of operation:
```

Author: Done

   }
   ```

## Build and Deploy the Application

> In the `Dockerfile`, you will see how the dependencies are installed using npm.

Reviewer: ```suggestion
In the `Dockerfile`, you can see how the dependencies are installed using npm.
```

Author: Done

  You can build and push this to your registry of choice via:

   ```shell
   docker build -t <image> .
   docker push <image>
   ```

> {{< tabs name="cloudevents_rust_deploy" default="kn" >}} {{% tab name="yaml" %}}

Reviewer 2: ```suggestion
{{< tabs name="cloudevents_nodejs_deploy" default="kn" >}} {{% tab name="yaml" %}}
```

> To deploy the Knative service, look in `service.yaml` and replace `<registry/repository/image:tag>` with the image just created.

Reviewer: ```suggestion
To deploy the Knative service, replace `<registry/repository/image:tag>` with the image just created in the `service.yaml` file.
```

Author: Done

## Removing the sample app deployment

To remove the sample app from your cluster, delete the service.

> {{< tabs name="cloudevents_rust_delete" default="kn" >}} {{% tab name="yaml" %}}

Reviewer 2: ```suggestion
{{< tabs name="cloudevents_nodejs_delete" default="kn" >}} {{% tab name="yaml" %}}
```
